SDBlockDevice.cpp

```diff
@@ -147,6 +147,11 @@
 #include "mbed_debug.h"
 #include <errno.h>
 
+/* Required version: 5.5.4 and above */
+#if (MBED_VERSION < MBED_ENCODE_VERSION(5,5,4))
```

Comment:
> I will add the check for MBED_VERSION and if not defined, will set warning (to verify version) instead macro. Does that sound good enough? In case of master, user will have to make sure that proper combination is picked.

I was also thinking about a warning. It should be OK, in most cases, ppl use a released version. If not, a warning should be OK. What others think? This might be a good to have an issue on mbed-os to gather feedback as this is a common problem - dependency on mbed-os, and master has no version.